src/wrappers/lambda_env.js

```diff
@@ -59,5 +60,5 @@ function getUserHandler() {
         );
     }
 }
-
-module.exports.wrapper = lambda.lambdaWrapper(getUserHandler());
+module.exports.wrapper = config.getConfig().isEpsagonDisabled ? getUserHandler() :
```

Comment:
the trinary operator will cause both expressions to be evaluated. better call `getUserHandler()` only once

Reply:
fixed